### ISSUE TITLE
Provide DropList's iteration index to its item slot

### ISSUE THREAD

**Issue by treystout** [rlemaigre/Easy-DnD#72]
I am drawing a user-ordered list of items and need to provide ordinal sequential numbers next to each item. For example if the user data is `[X, Y, Z]` I want the Drag objects in the list to look like this...

| things |
|-----------|
|(1) X|
|(2) Y|
|(3) Z|

and if the user drags Z to the top, I want it to look like this

| things |
|-----------|
|(1) Z|
|(2) X|
|(3) Y|

I think it would be as simple as just changing the following code
https://github.com/rlemaigre/Easy-DnD/blob/38adbc9a31fc4a1a75594a089d9e49daca3ffb21/lib/src/components/DropList.vue#L11

to...

```
<slot
  name="item"
  v-for="(item, index) in reorderedItems"
  :item="item"
  :index="index"
  :reorder="index === closestIndex"
/>
```

**Comment by treystout** [comment]
just noticed this is a dupe of #59, @rlemaigre would you prefer a PR? that would close this and #59?

**Comment by rlemaigre** [comment]
Hello,

I don't have much time to maintain this anymore. But I'm starting a new app for work that will be based on Vue 3. Then, I'll rewrite this project to make it compatible with Vue 3, and I'll start maintaining that instead. Hopefully by that time, I'll have learned how to properly unit test something like this. Proper tests are sorely lacking at the moment.

I still accept pull requests though ! If the change is small, there's a good chance I'll accept it.

Best regards

**Comment by rlemaigre** [comment]
The change is available in version 1.11.0.
